chapters/hpmor-chapter-017.tex

```diff
@@ -612,12 +612,11 @@ \chapter{Locating the Hypothesis}
 
 “It is not,” said Dumbledore, shaking his head and looking very serious. “I took this from the ruins of James and Lily’s home in Godric’s Hollow, where also I found you; and I have kept it from then until now, against the day when I could give it to you.”
 
-In the mixture of hypotheses that served as Harry’s model of the world, Dumbledore’s insanity was rapidly rising in probability. But there \emph{was} still a substantial amount of probability allocated to other alternatives…“Um, is it a \emph{magical} rock?”
+In the mixture of hypotheses that served as Harry’s model of the world, Dumbledore’s insanity was rapidly rising in probability. But there \emph{was} still a substantial amount of probability allocated to other alternatives… “Um, is it a \emph{magical} rock?”
```

Comment:
No space needed here.

Reply:
Really? I find it quite dense without, but I am no expert on typesetting...

Reply:
I think so! I agree about the look to an extent, but I think it's logical. (If it really needed fixing I'd rather just space the ellipsis a little more.)

Reply:
Ok, than I am gonna revert these changes.